Resources/Prototypes/GameRules/roundstart.yml

```diff
@@ -416,6 +416,7 @@
         tableId: BasicAntagEventsTable
       - !type:NestedSelector
         tableId: ModerateAntagEventsTable
+        weight: 0.5
```

Comment:
<img width="1662" height="337" alt="Image" src="https://github.com/user-attachments/assets/8c968680-5ad2-405f-a177-09c5a80ddfad" />
You can't, it's only used by `GroupSelector`.